cli_tools_tests/module/README.md

```diff
@@ -1,4 +1,4 @@
-This is an e2e test for `cli_tools/common/disk/inspect.go`.
+Module tests: Larger than unit tests, but smaller than e2e tests.
```

Comment:
E2E vs. module tests is not about size, but what is being tested. A module test can test the same behavior as unit test. In our case, the main difference is whether a whole module is tested using real data (module) or if anything is mocked (unit)

Reply:
Updated; see latst in https://github.com/GoogleCloudPlatform/compute-image-tools/pull/1381